doc/install_generic.txt

```diff
@@ -0,0 +1,110 @@
+.. raw:: html
+
+    <style> .code {background-color:#F8F8F8; font-family: "Consolas","Deja Vu Sans Mono","Bitstream Vera Sans Mono",monospace;} </style>
+    <style> .bcode {color:blue; background-color:#F8F8F8; font-family: "Consolas","Deja Vu Sans Mono","Bitstream Vera Sans Mono",monospace;} </style>
+    <style> .rcode {color:#F00; background-color:#F8F8F8; font-family: "Consolas","Deja Vu Sans Mono","Bitstream Vera Sans Mono",monospace;} </style>
+    <style> .blue {color:blue} </style>
+    <style> .red {color:red} </style>
+
+.. _install_generic:
+
+
+Generic Install Instructions
+============================
+
+If you want the latest on the edge installation of Theano, you want `bleeding_edge`_.
+Otherwise go for the `stable`_.
+
+.. _stable:
+
+Stable Installation
+------------------
+
+These are general instructions that will install the latest stable version of Theano.
+
+pip
+~~~
+
+The recommended way to obtain the latest stable version of Theano is from PyPI using
+pip_. 
+
+.. code-block:: bash
+
+    <sudo> pip install <--user> Theano[test] 
+
+.. role:: code
+
+.. role:: bcode
+
+.. role:: rcode
+
+:rcode:`sudo` :code:`pip install` :bcode:`--user` :code:`Theano[test]`
```

Comment:
I found several ways to get what we want with the colors, none of them is completely satisfactory.

I made a gist with a working example (you have to `include` the first 10 lines before you `include` the installation instructions in `install_ubuntu.txt`):
https://gist.github.com/fvisin/db7bbce7aed39cf1d9d6aee4fa644c1c

If writing in HTML is not too painful, I guess that the last option is the best. Otherwise we will have to disable `html_use_smartypants` in the configs, but I think we want it to be on in the rest of the documentation. Let's discuss it IRL.